README.md

```diff
@@ -16,10 +16,10 @@ The goal of this library is to provide a simple and intuitive approach to design
 
 ##### State 
 
-`State` is the single source of truth. It represents a state of your system and is usually a plain swift type (a type that does not contain any ReactiveSwift primitives). The only way to change from one `State` to another is to emit an `Event`
+`State` is the single source of truth. It represents a state of your system and is usually a plain Swift type (a type that does not contain any ReactiveSwift primitives). Your state is immutable. The only way to transition from one `State` to another is to emit an `Event`.
```

Comment:
`(a type that does not contain any ReactiveSwift primitives)` could be changed to `(which that does not contain any ReactiveSwift primitives)` to avoid repetition.

Reply:
I'm a bit confused? "which that does not" isn't valid grammar... I think this sentence is fine as is, but open to changing if you have a different though

Reply:
Sorry, I mistyped. I meant it without the `that`. `(which does not contain any ReactiveSwift primitives)`. But it's bike shedding.

Reply:
Makes sense, done!